relay-general/src/pii/redactions.rs

```diff
@@ -29,9 +29,25 @@ impl Default for ReplaceRedaction {
 }
 
 /// Defines how replacements happen.
+#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq)]
+#[serde(rename_all = "snake_case", untagged)]
+pub enum Redaction {
+    /// All the reduction methods currently supported.
+    Method(RedactionMethod),
+    /// All the rest. Added for forward compatibility.
+    #[serde(skip_serializing)]
+    Other(serde_json::Value),
```

Comment:
Could this not be a unit variant on the old `Redaction` enum?

```rust
pub enum Redaction {
    // ...
    Hash,
    
    #serde(other)
    Other,
}
```

IMO this would classify the variant as `Other` and ignore any other fields in the object.

Reply:
Yes. I still think we could actually log it or send the error in case that something strange comes in.  But all this definitely can be ignored if we do not care about this.

Reply:
👍  reverted all to additional `Other` variant, and added `warn!` if this variant ever used.